### PR TITLE
#16796 Not allow Push publish in receiver without license

### PR DESCRIPTION
**Pull request by freddyucv**

### CODE REVIEW

File: dotCMS/src/main/java/com/dotcms/rest/BundlePublisherResource.java [hunk 1]
```diff
@@ -62,6 +65,12 @@ public Response publish(
 			@Context final HttpServletRequest  request,
 			@Context final HttpServletResponse response
 	) throws Exception {
+
+
+		if (LicenseUtil.getLevel() == LicenseLevel.COMMUNITY.level) {
```
Comment (dsilvam) [inline dotCMS/src/main/java/com/dotcms/rest/BundlePublisherResource.java:70]: LicenseManager.getInstance().isCommunity()

File: dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java [hunk 2]
```diff
@@ -194,6 +197,10 @@ private boolean isJobRunning() {
     @Produces("text/plain")
     public Response generateIntegrityData(@Context HttpServletRequest request)  {
 
+        if (LicenseUtil.getLevel() == LicenseLevel.COMMUNITY.level) {
```
Comment (dsilvam) [inline dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java:200]: `LicenseManager.getInstance().isCommunity()`

File: dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java [hunk 3]
```diff
@@ -790,6 +797,11 @@ public Response fixConflictsFromRemote ( @Context final HttpServletRequest reque
                                              @FormDataParam("DATA_TO_FIX") InputStream dataToFix,
                                              @FormDataParam("TYPE") String type ) throws JSONException {
 
+
+        if (LicenseUtil.getLevel() == LicenseLevel.COMMUNITY.level) {
```
Comment (dsilvam) [inline dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java:801]: `LicenseManager.getInstance().isCommunity()`

File: dotCMS/src/main/java/com/dotcms/rest/BundlePublisherResource.java (hunk 1 above)
Comment (freddyucv) [reply dotCMS/src/main/java/com/dotcms/rest/BundlePublisherResource.java]: done https://github.com/dotCMS/core/pull/19488/commits/60cab2a324c4d93816bb55e733361c002d14a78c

File: dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java (hunk 2 above)
Comment (freddyucv) [reply dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java]: done https://github.com/dotCMS/core/pull/19488/commits/60cab2a324c4d93816bb55e733361c002d14a78c

File: dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java (hunk 3 above)
Comment (freddyucv) [reply dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java]: done https://github.com/dotCMS/core/pull/19488/commits/60cab2a324c4d93816bb55e733361c002d14a78c

**Pull request closed without merging**